<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1006</width>
    <height>594</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1006</width>
    <height>594</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1006</width>
    <height>594</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>dbsDriver</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">color:white;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="SpeedPanel" name="speedPanel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>58</y>
      <width>431</width>
      <height>431</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black</string>
    </property>
    <property name="text">
     <string>km/h</string>
    </property>
   </widget>
   <widget class="RpmPanel" name="rpmPanel">
    <property name="geometry">
     <rect>
      <x>509</x>
      <y>290</y>
      <width>251</width>
      <height>251</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black</string>
    </property>
    <property name="text">
     <string>1000/min</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonInfo">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>803</x>
      <y>280</y>
      <width>61</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #333;</string>
    </property>
    <property name="text">
     <string>Info</string>
    </property>
   </widget>
   <widget class="SwitchGearIndicator" name="sgiWidget" native="true">
    <property name="geometry">
     <rect>
      <x>728</x>
      <y>292</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="speedLcd">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>250</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;color:#333</string>
    </property>
    <property name="digitCount">
     <number>3</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Filled</enum>
    </property>
    <property name="value" stdset="0">
     <double>0.000000000000000</double>
    </property>
    <property name="intValue" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="InfoPanel" name="infoPanel" native="true">
    <property name="geometry">
     <rect>
      <x>531</x>
      <y>60</y>
      <width>401</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonNav">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>878</x>
      <y>280</y>
      <width>61</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #333;</string>
    </property>
    <property name="text">
     <string>Navi</string>
    </property>
   </widget>
   <widget class="ShadePanel" name="shadePanel" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>53</y>
      <width>441</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:black;</string>
    </property>
   </widget>
   <widget class="QLabel" name="gearLabel">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>400</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>OCR A Extended</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;color:#999</string>
    </property>
    <property name="text">
     <string>R</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonA">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>803</x>
      <y>330</y>
      <width>29</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #333;</string>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="StartButton" name="buttonStart">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>400</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #333;</string>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="ShadePanel" name="shadePanel2" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>503</x>
      <y>284</y>
      <width>263</width>
      <height>263</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:black;</string>
    </property>
   </widget>
   <widget class="ShadePanel" name="shadePanel3" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>525</x>
      <y>58</y>
      <width>413</width>
      <height>165</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:black;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonTrip">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>878</x>
      <y>330</y>
      <width>61</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #333;</string>
    </property>
    <property name="text">
     <string>Trip</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>222</x>
      <y>522</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Brush Script MT</family>
      <pointsize>40</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black</string>
    </property>
    <property name="text">
     <string>dbs Hot Drive</string>
    </property>
   </widget>
   <widget class="QLabel" name="dbsLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>520</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Brush Script MT</family>
      <pointsize>40</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #800033</string>
    </property>
    <property name="text">
     <string>dbs Hot Drive</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonHelp">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #333;</string>
    </property>
    <property name="text">
     <string>?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonM">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>834</x>
      <y>330</y>
      <width>29</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #333;color: #ff0066;</string>
    </property>
    <property name="text">
     <string>M</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAcc">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>536</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Sans Unicode</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#333;</string>
    </property>
    <property name="text">
     <string>accelerate</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelBra">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>462</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Sans Unicode</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#333;</string>
    </property>
    <property name="text">
     <string>brake</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSup">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>486</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Sans Unicode</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#333;</string>
    </property>
    <property name="text">
     <string>gear up</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSow">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>510</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Sans Unicode</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#333;</string>
    </property>
    <property name="text">
     <string>gear down</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelBra_2">
    <property name="geometry">
     <rect>
      <x>42</x>
      <y>463</y>
      <width>140</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Sans Unicode</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;</string>
    </property>
    <property name="text">
     <string>brake</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSow_2">
    <property name="geometry">
     <rect>
      <x>42</x>
      <y>511</y>
      <width>140</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Sans Unicode</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;</string>
    </property>
    <property name="text">
     <string>gear down</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSup_2">
    <property name="geometry">
     <rect>
      <x>42</x>
      <y>487</y>
      <width>140</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Sans Unicode</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;</string>
    </property>
    <property name="text">
     <string>gear up</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAcc_2">
    <property name="geometry">
     <rect>
      <x>42</x>
      <y>537</y>
      <width>230</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Sans Unicode</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;</string>
    </property>
    <property name="text">
     <string>accelerate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonTempoPlus">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>228</y>
      <width>29</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #333;</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonTempoMinus">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>288</y>
      <width>29</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #333;</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonTempoSetReset">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>258</y>
      <width>29</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #333;</string>
    </property>
    <property name="text">
     <string>SR</string>
    </property>
   </widget>
   <zorder>labelSup_2</zorder>
   <zorder>labelSow_2</zorder>
   <zorder>labelAcc_2</zorder>
   <zorder>labelBra_2</zorder>
   <zorder>shadePanel3</zorder>
   <zorder>shadePanel2</zorder>
   <zorder>shadePanel</zorder>
   <zorder>rpmPanel</zorder>
   <zorder>speedPanel</zorder>
   <zorder>buttonInfo</zorder>
   <zorder>sgiWidget</zorder>
   <zorder>infoPanel</zorder>
   <zorder>buttonNav</zorder>
   <zorder>gearLabel</zorder>
   <zorder>buttonA</zorder>
   <zorder>buttonStart</zorder>
   <zorder>speedLcd</zorder>
   <zorder>buttonTrip</zorder>
   <zorder>label</zorder>
   <zorder>dbsLabel</zorder>
   <zorder>buttonHelp</zorder>
   <zorder>buttonM</zorder>
   <zorder>labelAcc</zorder>
   <zorder>labelBra</zorder>
   <zorder>labelSup</zorder>
   <zorder>labelSow</zorder>
   <zorder>buttonTempoPlus</zorder>
   <zorder>buttonTempoMinus</zorder>
   <zorder>buttonTempoSetReset</zorder>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RpmPanel</class>
   <extends>QPushButton</extends>
   <header>rpmpanel.h</header>
  </customwidget>
  <customwidget>
   <class>SwitchGearIndicator</class>
   <extends>QWidget</extends>
   <header>switchgearindicator.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SpeedPanel</class>
   <extends>QPushButton</extends>
   <header>speedpanel.h</header>
  </customwidget>
  <customwidget>
   <class>InfoPanel</class>
   <extends>QWidget</extends>
   <header>infopanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StartButton</class>
   <extends>QPushButton</extends>
   <header>startbutton.h</header>
  </customwidget>
  <customwidget>
   <class>ShadePanel</class>
   <extends>QWidget</extends>
   <header>shadepanel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
